Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -180,24 +177,6 @@ bool GameListItem::IsValid() const
   return true;
 }
 
-void GameListItem::ReloadINI()
-{
-  if (!IsValid())
-    return;
-
-  IniFile ini = SConfig::LoadGameIni(m_game_id, m_Revision);
-  ini.GetIfExists("EmuState", "EmulationStateId", &m_emu_state, 0);
-  ini.GetIfExists("EmuState", "EmulationIssues", &m_issues, std::string());
-
-  m_custom_name.clear();
-  m_has_custom_name = ini.GetIfExists("EmuState", "Title", &m_custom_name);
```

Comment:
here is the the rather obscure "custom name via ini" feature that was removed. Not that no code actually loads any "titles.txt", even though the comments and variable names suggest that it's A Thing... (instead, it's just the result of `title_database.GetTitleName`, which is still here.